content/en/blog/_posts/XX-dra-consumable-capacity.md

```diff
@@ -0,0 +1,189 @@
+With DRA, users can request specific types of devices based on their attributes, define custom configurations tailored to their workloads, and even share the same resource among multiple containers or Pods.
+
+In this blog, we focus on the device sharing feature and dive into a new capability introduced in Kubernetes 1.34: `DRAConsumableCapacity`, which extends DRA to support finer-grained device sharing.
+
+## Background: Device Sharing via ResourceClaims
+
+From the beginning, DRA introduced the ability for multiple Pods to share a device by referencing the same ResourceClaim.
+This design decouples resource allocation from specific hardware, allowing for more dynamic and reusable provisioning of devices.
+
+In Kubernetes 1.33, the Partitionable Devices feature added support for resource drivers to advertise slices of a device that are available—rather than exposing the entire device as an all-or-nothing resource.
+This enabled Kubernetes to model shareable hardware more accurately.
+
+But there was still a missing piece: it doesn't yet support scenarios
+where the device driver manages fine-grained, dynamic portions of a device resource—like network bandwidth—based on user demand,
+or to share those resources independently of ResourceClaims, which are restricted by their spec and namespace.
+
+That’s where the DRAConsumableCapacity feature comes in.
+
+## 🎯 What Does the DRAConsumableCapacity Feature Unlock?
+
+### Device sharing across multiple ResourceClaims or DeviceRequests
+
+With the DRAConsumableCapacity feature, resource drivers can now support sharing the same device—or even a slice of a device—across multiple ResourceClaims or across multiple DeviceRequests.
+
+This means that Pods from different namespaces can simultaneously share the same device.
+
+### Device resource allocation
+
+The DRAConsumableCapacity feature extends the allocation algorithm in the scheduler to support allocating a portion of a device's resources as defined in the `capacity` field,
+and to ensure that the total allocated capacity across all consumers **never exceeds the device’s total capacity**, even when shared across multiple ResourceClaims or DeviceRequests.
+
+This feature expands support for scenarios where the device driver is able to manage resources *within* a device on a per-process basis—for example,
+allocating a specific amount of memory (e.g., 8 GiB) from a virtual GPU
+or setting bandwidth limits on virtual network interfaces allocated to specific Pods. This aims to provide safe and efficient resource sharing.
+
+### DistinctAttribute constraint
+
+In addition to the existing `MatchAttribute` constraint, this feature introduces a new constraint: `DistinctAttribute`.
+
+The primary goal of `DistinctAttribute` is to **prevent the same device from being allocated multiple times within a single `ResourceClaim`**. This ensures that each allocation refers to a distinct resource, even if they belong to the same device class.
```

Comment:
```suggestion
The primary goal of `DistinctAttribute` is to **prevent the same underlying device from being allocated multiple times within a single `ResourceClaim`**, which could happen since we are allocating shares (or subsets) of devices. This constraint ensures that each allocation refers to a distinct resource, even if they belong to the same device class.
```